src/css/components/panel.scss

```diff
@@ -166,6 +167,10 @@ h4 + .panel {
     margin-bottom: 0;
     padding-bottom: $panel-padding;
   }
+
+  .panel-row {
+    padding-bottom: 0;
+  }
```

Comment:
The padding on the bottom is causing the bottom border and icons to not have enough spacing. This impacting all `.panel-row-boxed .panel-row` 

![screen shot 2016-12-30 at 1 57 39 pm](https://cloud.githubusercontent.com/assets/1449852/21570877/15e14f74-ce98-11e6-9591-a9f9613d07a2.png)